src/main/kotlin/com/virtuslab/dokka/site/processors.kt

```diff
@@ -43,39 +28,27 @@ abstract class BaseStaticSiteProcessor(cxt: DokkaContext) : PageTransformer {
     val root = rawRoot ?: File("unknown")
 
     protected val mySourceSet = cxt.configuration.sourceSets.toSet()
+    protected val myDisplaySourceSet = mySourceSet.map { it.toDisplaySourceSet() }.toSet()
     protected val docsFile = File(root, "docs")
 
-    protected fun File.asDri(): DRI {
-        val relativePath = root.toPath().relativize(toPath()).toString().replace(File.separatorChar, '.')
-        return DRI("_.$relativePath")
+    data class LoadedTemplete(val template: TemplateFile, val children: List<LoadedTemplete>, val file: File) {
```

Comment:
`s/LoadedTemplete/LoadedTemplate/` and maybe rename the val `template` to avoid confusion?